client/components/takeQuiz/TakeQuiz.jsx

```diff
@@ -1,18 +1,32 @@
 import React, { useState, useEffect } from "react";
-import { Modal, Grid, Button, Box, FormControl, FormControlLabel, FormLabel, Radio, RadioGroup } from "@material-ui/core";
-import { makeStyles } from "@material-ui/core/styles";
-import { getSingleQuiz, submitQuizAnswers, getQuizGlobalRankings, getQuizFriendRankings } from "../../../api_master";
+import { Modal, Grid, Button, Box, FormControl, FormControlLabel, FormLabel, Radio, RadioGroup, Typography } from "@material-ui/core";
+import { makeStyles, withStyles } from "@material-ui/core/styles";
+import { getSingleQuiz, submitQuizAnswers, getQuizGlobalRankings, getQuizFriendRankings, getFriends } from "../../../api_master";
 import { useHistory, useParams } from "react-router-dom";
+import ChallengeFriend from '../friends/ChallengeFriend';
 
 const useStyles = makeStyles(theme => ({
+  root: {
+    fontSize: "20px",
+    color: "#303c6c",
+    fontWeight: "bold",
+  },
+  item: {
+    width: "90%"
+  },
+  page: {
+    '& Button': {
+      border: "2px solid #303c6c",
+      backgroundColor: "#d2fdff"
+    }
+  },
   quiz: {
     width: '100%',
-    backgroundColor: theme.palette.background.paper,
     alignContent: 'flex-start',
   },
   modal: {
     position: "absolute",
-    width: "50%",
+    width: "1000px",
```

Comment:
For future reference, if you wanna make the modal size more dynamic based on screen resolution I would try using vw/vh instead of px.

Reply:
Awesome, thank you! will keep this in mind.